diff --git a/third_party/cpu/runtime/cpu_runtime.cpp b/third_party/cpu/runtime/cpu_runtime.cpp
index 68b7efa7..193d8772 100644
--- a/third_party/cpu/runtime/cpu_runtime.cpp
+++ b/third_party/cpu/runtime/cpu_runtime.cpp
@@ -28,14 +28,14 @@ const int FLOAT_PREC = 4;
 const int ELEMS_PER_LINE = 8;
 
 using FLOAT16 = struct _FLOAT16 {
-#ifdef FLT16_MAX
+#ifdef FLT16_MAX_BOGUS
   _Float16 x;
 #else
   uint16_t x;
 #endif
 
   float toFloat32() const {
-#ifdef FLT16_MAX
+#ifdef FLT16_MAX_BOGUS
     return static_cast<float>(x);
 #else
     // Based on https://gist.github.com/zhuker/b4bd1fb306c7b04975b712c37c4c4075
